### DISCUSSION TITLE
Update collection "Translation" through API

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15642] [Archive]
Hi,

I need some help with updating a "Translations" collection through API. Is it possible to direct update this collection with API? Or do I need to update Translations collection through my Article collection which have this "Translations"?

Best regards,
Tony

**Comment by br41nslug** [comment]
Translations are M2M relations under the hood so this documentation about saving relational data should be a good place to start 😄 
https://docs.directus.io/reference/introduction.html#one-to-many-many-to-many
